components/core/src/clp/CurlDownloadHandler.cpp

```diff
@@ -56,6 +60,27 @@ CurlDownloadHandler::CurlDownloadHandler(
         m_http_headers.append("Cache-Control: no-cache");
         m_http_headers.append("Pragma: no-cache");
     }
+    std::set<std::string> const kReservedHeaders = {"range", "cache-control", "pragma"};
```

Comment:
In terms of naming, our guideline is:
1. If it's a compile-time constant, we should use `cXXX` in Camel case, not `kXXX`
2. The rule should only apply to a compile-time constant, normally should start with `constexpr`. This line is not a compile-time constant (but just an immutable runtime variable), so 1 is not applicable.

Reply:
fixed in next commit